types/11ty__eleventy-plugin-rss/11ty__eleventy-plugin-rss-tests.ts

```diff
@@ -0,0 +1,38 @@
+import pluginRss = require("@11ty/eleventy-plugin-rss");
+import { EleventyConfig } from "11ty.ts";
+import { closingSingleTagOptionEnum } from "posthtml-render";
```

Comment:
This is a dev only dep; do you actually need it? That'd obviate https://github.com/microsoft/DefinitelyTyped-tools/pull/1104

Reply:
Er, no, sorry, my ctrl+f failed me